CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
   - [#443](https://github.com/Datatamer/tamr-client/pull/443) Added function to materialize datasets.
   - [#445](https://github.com/Datatamer/tamr-client/pull/445) Added functions for getting projects and datasets by name via `tc.project.by_name` and `tc.dataset.by_name`
     - Renamed functions `from_resource_id` to `by_resource_id` in `tc.attribute`, `tc.dataset`, `tc.operation`, and `tc.project`
+  - [#446](https://github.com/Datatamer/tamr-client/pull/446) Added functions for categorization workflow operations in `tc.categorization` and schema mapping workflow operations in `tc.schema_mapping`
+    - Moved function `tc.categorization.project.manual_labels` to `tc.categorization.manual_labels`
```

Comment:
nit: Since `manual_labels` is new since last release, we can squash the entries in the changelog.